apps/supernova/src/helpers.js

```diff
@@ -4,37 +4,34 @@
  */
 
 export const parseError = (error) => {
-  let errMsg = error
-
   // check if error is JSON containing message or just string
   if (typeof error === "string") {
-    errMsg = parseMessage(error)
+    if (error === "Failed to fetch") {
```

Comment:
I recommend not doing a hard equality check here but a more soft substring check. It's more robust if the error message changes slightly.
```suggestion
    if (error.includes("Failed to fetch")) {
```